libr/bin/bfile.c

```diff
@@ -82,6 +91,11 @@ static int string_scan_range(RList *list, RBinFile *bf, int min,
 		free (buf);
 		return -1;
 	}
+	RBinSection *s = r_bin_get_section_at (bf->o, from, false);
```

Comment:
why `false` for the `va` parameter? it seems you are using the `s->vaddr` in the next lines, so you are probably dealing with virtual addresses...

Reply:
nope those are phisical addresses. some string sections are not virtually mapped.

Reply:
But you are using s->vaddr few lines below this...